Comp2025/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -526,6 +527,6 @@ public void autoInit( )
    */
   public void teleopInit( )
   {
-    // CommandScheduler.getInstance( ).schedule(m_elevator.getCalibrateCommand( ));
+    CommandScheduler.getInstance( ).schedule(m_elevator.getCalibrateCommand( ));
```

Comment:
Calibration for 2024 involved running the climber down (winch the rope down) until the current limit shoots up to detect when we're hitting the hard stop--then reset the internal TalonFX encoder. For 2025, we don't want to do this. Calibration will involve sensing a Hall Effect limit switch when the elevator is full down and zeroing the TalonFX internal encoder. Bottom line, this will probably not be a command at startup, and we should remove this scheduled command.